sympy/physics/units.py

```diff
@@ -260,6 +260,7 @@ def free_symbols(self):
 avogadro_number = Rational('6.022140857') * 10**23
 avogadro = avogadro_constant = avogadro_number / mol
 boltzmann = Rational('1.38064852') * ten**-23 * J / K
+stefan_boltzman = Rational('5.670367')*ten**-8*W*m**-2*K**-4
```

Comment:
It should be boltzmann. I didnot understand your second comment. Please clarify @smichr .